aperture mask unit test fails due to precision

On some builds `test_mask_multiply()` and `test_mask_multiply_quantity()` fail because of float machine precision

See for example the current [build log for i586 in the OBS devel package](https://build.opensuse.org/package/live_build_log/devel:languages:python:numeric/python-photutils/openSUSE_Tumbleweed/i586):
```
[  619s] =================================== FAILURES ===================================
[  619s] ______________________________ test_mask_multiply ______________________________
[  619s] 
[  619s]     def test_mask_multiply():
[  619s]         radius = 10.
[  619s]         data = np.ones((50, 50))
[  619s]         aper = CircularAperture((25, 25), r=radius)
[  619s]         mask = aper.to_mask()
[  619s]         data_weighted = mask.multiply(data)
[  619s] >       assert np.sum(data_weighted) == radius**2 * np.pi
[  619s] E       assert 314.15926535897927 == ((10.0 ** 2) * 3.141592653589793)
[  619s] E        +  where 314.15926535897927 = <function sum at 0xb120526c>(array([[0.        , 0.        , 0.        , 0.        , 0.        ,\n        0.        , 0.        , 0.0590655 , 0.2935...271 , 0.0590655 , 0.        ,\n        0.        , 0.        , 0.        , 0.        , 0.        ,\n        0.        ]]))
[  619s] E        +    where <function sum at 0xb120526c> = np.sum
[  619s] E        +  and   3.141592653589793 = np.pi
[  619s] 
[  619s] photutils/aperture/tests/test_mask.py:113: AssertionError
[  619s] _________________________ test_mask_multiply_quantity __________________________
[  619s] 
[  619s]     def test_mask_multiply_quantity():
[  619s]         radius = 10.
[  619s]         data = np.ones((50, 50)) * u.adu
[  619s]         aper = CircularAperture((25, 25), r=radius)
[  619s]         mask = aper.to_mask()
[  619s]     
[  619s]         data_weighted = mask.multiply(data)
[  619s]         assert data_weighted.unit == u.adu
[  619s] >       assert np.sum(data_weighted.value) == radius**2 * np.pi
[  619s] E       assert 314.15926535897927 == ((10.0 ** 2) * 3.141592653589793)
[  619s] E        +  where 314.15926535897927 = <function sum at 0xb120526c>(array([[0.        , 0.        , 0.        , 0.        , 0.        ,\n        0.        , 0.        , 0.0590655 , 0.2935...271 , 0.0590655 , 0.        ,\n        0.        , 0.        , 0.        , 0.        , 0.        ,\n        0.        ]]))
[  619s] E        +    where <function sum at 0xb120526c> = np.sum
[  619s] E        +    and   array([[0.        , 0.        , 0.        , 0.        , 0.        ,\n        0.        , 0.        , 0.0590655 , 0.2935...271 , 0.0590655 , 0.        ,\n        0.        , 0.        , 0.        , 0.        , 0.        ,\n        0.        ]]) = <Quantity [[0.        , 0.        , 0.        , 0.        , 0.        ,\n            0.        , 0.        , 0.0590655 ...655 , 0.        ,\n            0.        , 0.        , 0.        , 0.        , 0.        ,\n            0.        ]] adu>.value
[  619s] E        +  and   3.141592653589793 = np.pi
[  619s] 
[  619s] photutils/aperture/tests/test_mask.py:128: AssertionError
[  619s] ======== 2 failed, 1249 passed, 24 skipped, 2 xfailed in 526.56 seconds ========
```

--- a/photutils/aperture/tests/test_mask.py
+++ b/photutils/aperture/tests/test_mask.py
@@ -5,7 +5,7 @@ Tests for the mask module.
 
 import astropy.units as u
 import numpy as np
-from numpy.testing import assert_allclose
+from numpy.testing import assert_allclose, assert_almost_equal
 import pytest
 
 from ..bounding_box import BoundingBox
@@ -110,7 +110,7 @@ def test_mask_multiply():
     aper = CircularAperture((25, 25), r=radius)
     mask = aper.to_mask()
     data_weighted = mask.multiply(data)
-    assert np.sum(data_weighted) == radius**2 * np.pi
+    assert_almost_equal(np.sum(data_weighted), radius**2 * np.pi)
 
     # test that multiply() returns a copy
     data[25, 25] = 100.
@@ -125,7 +125,7 @@ def test_mask_multiply_quantity():
 
     data_weighted = mask.multiply(data)
     assert data_weighted.unit == u.adu
-    assert np.sum(data_weighted.value) == radius**2 * np.pi
+    assert_almost_equal(np.sum(data_weighted.value), radius**2 * np.pi)
 
     # test that multiply() returns a copy
     data[25, 25] = 100. * u.adu

